src/pages/MarketPlacePage/MarketPlaceTaskDetail.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+
+import projectData from "src/data/dummy_data_extended.json";
+import { slugify } from "@/src/utils/string-utils";
+
+import Tag from "@/src/components/Tag/Tag";
+import "./MarketPlaceTaskDetail.scss";
+import TaskPosterCard from "./components/TaskPosterCard/TaskPosterCard";
+
+function MarketplaceTaskDetail() {
+  const { id } = useParams();
+  const [jobs, setJobs] = useState([]);
+  const [jobDetail, setJobDetail] = useState();
```

Comment:
This is an unused var. I would link up the render with the properties of job detail instead of hard coding it. You should activate a warning for an unused var.

Note this ticket is sorely lacking details and therefore scope of this ticket would probably confuse you https://makeitmvp.atlassian.net/jira/software/c/projects/LAP4SE/issues/LAP4SE-104?jql=project%20%3D%20%22LAP4SE%22%20ORDER%20BY%20created%20DESC

Also, you need to adjust for placeholder messages for no matches and for if there's no data loaded.